test.js

```diff
@@ -85,3 +85,45 @@ test('add tasks', t => {
 
 	t.is(list._tasks.length, 4);
 });
+
+test('skip tasks', t => {
+	t.plan(5);
+
+	const list = new Listr([
+		{
+			title: 'Task 1',
+			task: () => new Listr([
+				{title: 'Task 1.1', task: () => t.pass() || '1.1'},
```

Comment:
> I had the t.plan(5) and the t.pass() to ensure that all the non-skipped tasks still ran.

That's a valid point :). But can you change it to use `async`/`await` then? You can keep the `t.plan(5)` as that indeed makes sense.

> currently the implementation of Listr requires the task() function return a result

That indeed was the case but I fixed that yesterday as you can see [here](https://github.com/SamVerschueren/listr/blob/master/lib/task.js#L17-L19). It doesn't fail anymore if `result` is `undefined. So I believe it should work without the `|| '1.1';` stuff. If not, let me know because that should be fixed then.

> But as mentioned in the PR description I'm keen to give the tests some love, this was just to prove the concept really.

Very much appreciated :)!